### DISCUSSION TITLE
KEDA  Version and Operator logs for Azure Container Apps

### DISCUSSION THREAD

**Discussion by andreaskasc** [kedacore/keda#4325] [General] [answered]
Hi Everyone,

I have been experimenting with Apache Kafka Scaler on an Azure Container App and I am not getting the expected behavior. Before creating a bug I would like to get the full logs available to see what's going so I can better investigate/describe my issue. I can run the below query in Log Analytics workspace
**ContainerAppSystemLogs_CL
| where EventSource_s has "KEDA"**
and I get some logs but I don't get details on them, eg I get a warning "ScaledObject doesn't have correct scaleTargetRef specification" but no further details on it. I know the scaler can connect to the Azure Event Hub workspace (which has Kafka enabled - that's why I used Kafka scaler) and it works but not always as expected.

I would also like to know if Azure Container Apps are using the latest scaler version because by searching I saw that one of the unexpected behaviors I have is connected to an old issue (https://github.com/kedacore/keda/issues/2612) that should be already fixed and merged in latest version.

Thank you,
Andreas

**Comment by tomkerkhove** [comment, marked as answer]
Great to see you are using KEDA! However, Azure Container Apps is a Microsoft product and I recommend filing questions/issues on https://github.com/microsoft/azure-container-apps as this is purely for open-source KEDA.

For example, KEDA info will be shown soon: https://github.com/microsoft/azure-container-apps/issues/586

**Reply by andreaskasc** [reply]
Thank you @tomkerkhove. Will do that.
